### PR TITLE
The summary calculator should return zero PI for non flowing wells

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

File: src/opm/output/eclipse/Summary.cpp [hunk 1]
```diff
@@ -971,28 +971,50 @@ inline quantity potential_rate( const fn_args& args ) {
 
 inline quantity preferred_phase_productivty_index(const fn_args& args) {
     if (args.schedule_wells.empty())
-        return potential_rate<rt::productivity_index_oil>(args);
+        return {0, rate_unit<rt::productivity_index_oil>()};
```
Comment (bska) [inline src/opm/output/eclipse/Summary.cpp:974]: Strictly speaking, `potential_rate` will return the appropriate zero value in this case.

Comment (joakim-hove) [reply src/opm/output/eclipse/Summary.cpp:974]: I agree - but when we were already returning explicit zeros I think it became clearer this way. No strong feelings though.

**Review by bska — APPROVED** [review]
Okay, with the style consistency justification I think this looks good.  I'll merge into master.

**Comment by akva2** [comment]
This broke the post builder (no pre build performed..)

**Comment by joakim-hove** [comment]
> This broke the post builder (no pre build performed..)

Hmmm - the "jenkins build this please" ocd did not kick. Sorry. Will revert and then redo.

**Pull request merged**